Bottom: d3a4e0e052ef5f338e9a5ce35244ca5f8cfdb05b
Top:    e17f688847185891178b95b27c0e06823c6bb9d4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:23:16 +0000

Refresh of fixup-searchobjectbylocation

---

diff --git a/src/search.c b/src/search.c
index f0cb44b..bf7413a 100644
--- a/src/search.c
+++ b/src/search.c
@@ -1159,7 +1159,7 @@ SearchLayerObjectInternal (LayerType *layer,
       && SearchLineByLocation (locked, (LayerType **)Result1, (LineType **) Result2, (LineType **) Result3))
     return LINE_TYPE;
 
-    if ((HigherAvail & (PIN_TYPE | PAD_TYPE)) == 0 &&
+  if ((HigherAvail & (PIN_TYPE | PAD_TYPE)) == 0 &&
       Type & ARCPOINT_TYPE &&
       SearchArcPointByLocation (locked, (LayerType **)Result1, (ArcType **) Result2, (PointType **) Result3))
     return ARCPOINT_TYPE;
